### PR TITLE
Qadir Assignment7 logo change, textarea and add data textbox dummy added.

### PR DESCRIPTION
**Pull request by MKadeer17**
…ded.

### CODE REVIEW

File: Qadir__Work/HTML/Assignment7/Education.css [hunk 1]
```diff
@@ -74,4 +77,12 @@ input[type="button"] {
...
     cursor: pointer;
+}
+
+textarea  {
+    resize: none;
+}
+
+#add {
+    cursor: pointer;
 }
```
Comment (omerana14) [inline Qadir__Work/HTML/Assignment7/Education.css:88]: Add blank space here

File: Qadir__Work/HTML/Assignment7/Register.css [hunk 2]
```diff
@@ -62,4 +65,17 @@ body {
...
+}
+
+textarea {
+    padding: 2px;
+    resize: none;
+}
+
+.edu {
+    color: rgb(49, 49, 49) !important;
 }
```
Comment (omerana14) [inline Qadir__Work/HTML/Assignment7/Register.css:81]: here too

**Review by dnyaneshwarkhule — APPROVED** [review]

**Review by omerana14 — APPROVED** [review]

**Review by priya-ughade — APPROVED** [review]

File: Qadir__Work/HTML/Assignment7/Register.css (hunk 2 above)
Comment (MKadeer17) [reply Qadir__Work/HTML/Assignment7/Register.css:81]: Already Added

File: Qadir__Work/HTML/Assignment7/Education.css (hunk 1 above)
Comment (MKadeer17) [reply Qadir__Work/HTML/Assignment7/Education.css:88]: already added

**Pull request merged**